changelog/issue-6014.md

```diff
@@ -0,0 +1,5 @@
+audience: users
+level: patch
```

Comment:
this could also be a breaking change, if some of the existing tasks would start failing?

Reply:
or maybe not.. I wonder if we track somewhere payload validation errors

Reply:
I thought about this too, but decided since we have officially deprecated Docker Worker, and anything that is submitting such payloads are already buggy and should probably be fixed, it is reasonable to argue that it is a bugfix and that docker worker wasn't working correctly, since in any case the conversion mechanism of non-strings to strings wasn't documented or referenced, and as undocumented behaviour, under no contract of behaviour, is free to change.

Reply:
Spotted one from wpt project: https://community-tc.services.mozilla.com/tasks/HsL1bxj8SAWUtzqp_vUXeA

Reply:
And one more https://community-tc.services.mozilla.com/tasks/PUB9BKR1RSCJXnbZDiEN8w Project Orion